src/main/java/core/utils/Log.java

```diff
@@ -0,0 +1,48 @@
+package core.utils;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+
+/**
+ * This class is uses for .
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public final class Log {
+    private static final String LOG4J_PROPERTIES = "log4j.properties";
+    private static Log instance = null;
```

Comment:
The instance do not have to be instanced.

Reply:
Solved